api/app/Console/Kernel.php

```diff
@@ -18,14 +20,30 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
+        // Log facade does not seem to work in here!
+        // Config utility does not seem to work in here!
+
         // $schedule->command(HardDeleteOldUsers::class)->dailyAt('08:00');
 
-        // clean up old user generated files every day at 2:00 AM
+        // clean up old user generated files every day at 1:00 AM
         $schedule->command(PruneUserGeneratedFiles::class)
             ->timezone('America/Toronto')
-            ->dailyAt('2:00')
+            ->dailyAt('1:00')
             ->withoutOverlapping()
             ->appendOutputTo('/tmp/laravel-prune-user-generated-files.log');
+
+        if (config('feature.notifications')) {
+
+            // queue up Application Deadline Approaching emails every day, close to the time the pool would close
+            $schedule->command(SendNotificationsApplicationDeadlineApproaching::class)
+                ->timezone('America/Toronto')
+                // 10 PM Eastern is the same day across the country, close to the end of the day in NL
+                ->dailyAt('22:00')
```

Comment:
I ended up scheduling this daily instead of hourly like the issue described. Daily is enough to reliably run this without missing any pools. Also, if this runs hourly and is slightly off, you risk notifying a pool twice or not at all.

Gray confirmed that near the end-of-day in Newfoundland (when it is the same day across the country) is a good time to send the email.